src/decaylanguage/decay/decay.py

```diff
@@ -446,7 +447,8 @@ def __str__(self) -> str:
 
 
 Self_DecayChain = TypeVar("Self_DecayChain", bound="DecayChain")
-DecayModeDict = Dict[str, List[Dict[str, Union[float, str, List[Any]]]]]
+DecayModeDict = dict[str, Union[float, str, list[Any]]]
```

Comment:
Using dict and list won't work while we support Python 3.8. We will soon-ish drop support, but not just yet. You need to use Dict, List, etc.

Reply:
I'll come back to this, but I think I encountered the lower-case/builtin ones in pre-existing code?

Reply:
Right, we're OK as long as the type checking is dome with 3.9+. Fine, then.